lib/jestPreprocessor.js

```diff
@@ -0,0 +1,11 @@
+const babel = require('babel-core')
+const jestPreset = require('babel-preset-jest')
+
+module.exports = {
+  process: function (src) {
+    const transformCfg = {
+      presets: ['es2015', 'react', 'stage-0', jestPreset]
```

Comment:
Does this work if you pass the presets directly? I found the automatic lookup fails sometimes so `require`ing the presets and passing the modules directly would be ideal.